README.md

```diff
@@ -112,99 +172,140 @@ The job will appear on the WebUI and you can inspect/restart/kill it.
   src="https://cloud.githubusercontent.com/assets/617111/9421006/17bceb36-483a-11e5-8845-bb2234697a14.gif">
 </div>
 
+
 If you don't like Web UI's you can use the text-based UI:
 
-    ketrew interact
+    $ ketrew interact
+    [ketrew]
+        Main menu
+        Press a single key:
+        * [q]: Quit
+        * [v]: Toggle verbose
+        * [s]: Display current status
+        * [l]: Loop displaying the status
+        * [k]: Kill targets
+        * [e]: The Target Explorer™
+
+Finally to stop the server:
 
+    $ ketrew stop -P daemon
+    [ketrew] Server killed.
+
+As you can see, just from the command line, `ketrew submit` provides a power and flexible mechanism for launching tasks. But to go further we need to use an EDSL.
 
 The EDSL: Defining Workflows
 ----------------------------
 
-The previous section uses `ketrew submit` to launch an extremely simple
-workflow, to go further we need the EDSL.
-
 ### Overview
 
-The EDSL is an OCaml library where all the functions are used to build a
-workflow data-structure. Then, one function: `Ketrew.Client.submit` is used to
-submit workflows to the engine.
+The EDSL is an OCaml library where functions are used to build a
+workflow data-structure. Then, one function:
+[Ketrew.Client.submit](http://seb.mondet.org/software/ketrew/api/Ketrew.Client.html#VALsubmit)
+is used to submit workflows to the engine.
 
 A workflow is a Graph of “**targets**”.
 
 There are 3 kinds of links between targets:
 
-- *dependencies:* targets that need to be ensured or
+- *dependencies:* targets that need to be ensured to exist or
 run before a target can start,
-- *fallbacks:* targets that will be activated
-if the target fails, and
-- *success-triggers:* targets that will be activated only *after* a target
-succeeds.
+- *fallbacks:* targets that will be activated if the target fails, and
+- *successes:* targets that will be activated only *after* a target succeeds.
```

Comment:
yes but a "fallback" means an operation in case of **failure**, calling the other “success” is very weird → “a *success* that fails” sounds absurd.

we can just use the name of the argument `depends_on`, `on_success_activate`, etc.